.gitignore

```diff
@@ -1,2 +1,3 @@
 **/.DS_Store
-**/node_modules
\ No newline at end of file
+**/node_modules
+package-lock.json
```

Comment:
Also to actually remove package-lock.json from being tracked, run git rm —cache package-lock.json

Reply:
@shannonphu I tried to run it but it said "fatal: pathspec '—cache' did not match any files"

Reply:
Hmm make that gitignore line be **/package-lock.json. 
And I guess just remove the package-lock.json for now.